Application/EdFi.Ods.Api/Authorization/AuthorizationPolicyBuilderExtension.cs

```diff
@@ -0,0 +1,20 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+#if NETCOREAPP
+using Microsoft.AspNetCore.Authorization;
+
+namespace EdFi.Ods.Api.Authorization
```

Comment:
this should be in namespace EdFi.Ods.Api.Extensions

Reply:
fixed